frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/moderation/ModerationPortlet.java

```diff
@@ -128,7 +131,19 @@ private void renderActionView(RenderRequest request, RenderResponse response) th
         }
     }
 
-    private void acceptModerationRequest(User user, User requestingUser, ModerationRequest moderationRequest) throws TException {
+    private void declineModerationRequest(User user, ModerationRequest moderationRequest, RenderRequest request) throws TException {
+        switch (moderationRequest.getDocumentType()) {
+            case USER: {
```

Comment:
I don't think, that these braces are needed.
It might also be good to correct the indentations of the `break` statements.